content/boards/arduino-mega-2560/index.md

```diff
@@ -16,6 +16,17 @@ The Mega 2560 Rev3 is a classic Arduino board with a large number of IO pins and
 - 54 digital IO pins, 15 with PWM support.
 - 16 analog inputs (can be used as digital inputs).
 
+| Device | Limit | Notes |
+| ------------- | :-----------: | ----------- |
+| [Analog inputs](/devices/potentiometer/) | 16 | Potentiometers or linear hall sensors can be connected. |
+| [Input shift registers](/devices/input-shift-register/) | 6 | Six *chains* of shift registers, 32 bits each (2x16 or 4x8 bit shift registers). |
+| [Buttons](/devices/button-switch/) | 68 | |
+| [Encoders](/devices/encoder/) | 20 | |
+| [LCD screens](/devices/lcd-display/) | 2 | |
+| 7-segment displays | 6 | 6 [TM1637](/devices/seven-segment-tm1637/)modules, or 6 chains of [MAX7219](/devices/seven-segment-max7219/) modules, or a mix of both. You need external 5V power supply for more than one. |
```

Comment:
I've found myself avoiding "you" in the documentation, we should probably make it a style guidance. How about:

"A 5V power supply is required when connecting more than one display."